doc/faq/installing_faq.rst

```diff
@@ -42,127 +34,46 @@ If you are still having trouble, see :ref:`reporting-problems`.
+1. Delete the caches from your :ref:`Matplotlib configuration directory
    <locating-matplotlib-config-dir>`.
 
-2. Delete the ``build`` directory in the source tree.
-
-3. Delete any matplotlib directories or eggs from your :ref:`installation
+2. Delete any Matplotlib directories or eggs from your :ref:`installation
    directory <locating-matplotlib-install>`.
 
-How to Install
-==============
-
-.. _install-from-git:
-
-Source install from git
------------------------
-
-Clone the main source using one of::
-
-   git clone git@github.com:matplotlib/matplotlib.git
-
-or::
-
-   git clone git://github.com/matplotlib/matplotlib.git
-
-and build and install as usual with::
-
-  > cd matplotlib
-  > python setup.py install
-
-.. note::
-
-    If you are on debian/ubuntu, you can get all the dependencies
-    required to build matplotlib with::
-
-      sudo apt-get build-dep python-matplotlib
-
-    If you are on Fedora/RedHat, you can get all the dependencies
-    required to build matplotlib by first installing ``yum-builddep``
-    and then running::
-
-       su -c "yum-builddep python-matplotlib"
-
-    This does not build matplotlib, but it does get all of the
-    build dependencies, which will make building from source easier.
-
-
-If you want to be able to follow the development branch as it changes
-just replace the last step with (make sure you have **setuptools**
-installed)::
-
-  > python setup.py develop
-
-This creates links in the right places and installs the command
-line script to the appropriate places.
-
-.. note::
-   Mac OSX users please see the :ref:`build_osx` guide.
-
-   Windows users please see the :ref:`build_windows` guide.
-
-Then, if you want to update your matplotlib at any time, just do::
-
-  > git pull
-
-When you run `git pull`, if the output shows that only Python files have been
-updated, you are all set. If C files have changed, you need to run the `python
-setup.py develop` command again to compile them.
-
-There is more information on :ref:`using git <using-git>` in
-the developer docs.
-
-
 Linux Notes
 ===========
 
-Because most Linux distributions use some sort of package manager,
-we do not provide a pre-built binary for the Linux platform.
-Instead, we recommend that you use the "Add Software" method for
-your system to install matplotlib. This will guarantee that everything
-that is needed for matplotlib will be installed as well.
+To install Matplotlib at the system-level we recommend that you use your 
+distribution's package manager to install Matplotlib.  This will guarantee 
+that Matplotlib's dependencies will be installed as well.
 
-If, for some reason, you can not use the package manager, Linux usually
-comes with at least a basic build system. Follow the :ref:`instructions
-<install-from-git>` found above for how to build and install matplotlib.
+If, for some reason, you can not use the package manager, you may use the
+manylinux wheels available on PyPI::
```

Comment:
OK, when I read this I thought that `manylinux` was a typo.  Put in quotes?

Reply:
https://github.com/pypa/manylinux They never use quotes, I do not think we want to deviate from their notation.

Reply:
It's a bit jargon-esque but I'd prefer leaving it as it is.

Reply:
Not a big deal, but how about "...you may use the wheels available from manylinux, available on PyPI"

Reply:
`use the manylinux-tagged wheels, available on PyPI` OK?

Reply:
Sounds good!

Reply:
Upon further throught I just dropped the "manylinux", as they are the only kind of linux wheels on pypi anyways, and either the reader knows what it means in which case it's obvious the wheels should be manylinux-tagged, or the reader doesn't and that doesn't help him at all.